<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiViewWindowWidget</class>
 <widget class="QWidget" name="MultiViewWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>976</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="m_pPushButtonMax">
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Max.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>817</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="m_pPushButtonClose">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_pLabelName">
     <property name="text">
      <string>Window Name</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
